### PR TITLE
Feature/clearreportpage

### PR DESCRIPTION
**Pull request by andyquaranto**
More great stuff hacked together by me to review

### CODE REVIEW

File: EmployeeChatBot/appsettings.json [hunk 1]
```diff
@@ -6,20 +6,22 @@
...
   "EmailClient": {
-    "Host": "",
-    "Port": ""
+    "Host": "mail.transcat.com",
+    "Port": "25"
   },
 
   "ExceptionEmail": {
-    "From": "andy.quaranto@transcat.com",
+    "From": "coivd-health-assessment@transcat.com",
```
Comment (ecgifford) [inline EmployeeChatBot/appsettings.json]: typo

File: EmployeeChatBot/appsettings.json [hunk 2]
```diff
@@ -33,5 +35,9 @@
     "URL": "transinc.transmation.com",
     "Port": 636,
     "DirectoryClasses": ""
+  },
+
+  "HrUsers": {
+    "Users": ["jenlson", "angela.spalding", "eflood"]
```
Comment (ecgifford) [inline EmployeeChatBot/appsettings.json]: jnelson typo

Comment (andyquaranto) [reply EmployeeChatBot/appsettings.json]: Fixed

File: EmployeeChatBot/appsettings.json (hunk 1 above)
Comment (andyquaranto) [reply EmployeeChatBot/appsettings.json]: fixed

**Pull request merged**